hooks/tk-multi-reviewsubmission/preprocess_nuke.py

```diff
@@ -19,15 +19,16 @@ def get_processed_script(self, nuke_script_path, **kwargs):
             # TODO: publisher.util.get_publish_name() does this much better
             replace_data["file_base_name"] = os.path.basename(replace_data["path"]).split('.')[0]
 
-        if self.context.entity:
-            sg_entity_type = self.context.entity["type"]
-            sg_filters = [["id", "is", self.context.entity["id"]]]
+        context = self.parent.context
+        if context:
+            sg_entity_type = context.entity["type"]
+            sg_filters = [["id", "is", context.entity["id"]]]
```

Comment:
Need to ensure `context.entity` is valid before requesting its `type` attribute. If this is a Project-level context, for example, `entity == None`.

Reply:
Oops. It was checking that earlier. Fixed.